public/video-ui/src/actions/VideoActions/videoPageCreate.js

```diff
@@ -8,14 +10,6 @@ function requestVideoPageCreate() {
   };
 }
 
-function receiveVideoPageCreate(capiPage) {
-  return {
-    type: 'VIDEO_PAGE_CREATE_POST_RECEIVE',
-    newPage: capiPage,
-    receivedAt: Date.now()
-  };
-}
-
 function errorReceivingVideoPageCreate(error) {
```

Comment:
Can we use the showError action to replace this and slim the code down a little?